Src/Web/Web.Shared.Net/RequestTrackingTelemetryModule.cs

```diff
@@ -415,8 +416,24 @@ private class ChildRequestTrackingSuppressionModule
             /// <summary>
             /// Using this as a hash-set of current active requests. The second value is not used.
             /// </summary>
-            private static ConcurrentDictionary<string, bool> activeRequestsA = new ConcurrentDictionary<string, bool>(System.Environment.ProcessorCount, MAXSIZE);
-            private static ConcurrentDictionary<string, bool> activeRequestsB = new ConcurrentDictionary<string, bool>(System.Environment.ProcessorCount, MAXSIZE);
+            private static ConcurrentDictionary<string, bool> activeRequestsA = new ConcurrentDictionary<string, bool>(System.Environment.ProcessorCount, DEFAULTMAXVALUE);
+            private static ConcurrentDictionary<string, bool> activeRequestsB = new ConcurrentDictionary<string, bool>(System.Environment.ProcessorCount, DEFAULTMAXVALUE);
+
+            /// <summary>
+            /// Initializes a new instance of the <see cref="ChildRequestTrackingSuppressionModule" /> class.
+            /// </summary>
+            /// <param name="maxRequestsTracked">The maximum number of active requests to be tracked before resetting the dictionary.</param>
+            internal ChildRequestTrackingSuppressionModule(int maxRequestsTracked = DEFAULTMAXVALUE)
+            {
+                this.MAXSIZE = maxRequestsTracked > 0 ? maxRequestsTracked : DEFAULTMAXVALUE;
+
+                System.Diagnostics.Debug.WriteLine($"{nameof(RequestTrackingTelemetryModule)}.{nameof(ChildRequestTrackingSuppressionModule)} Initialized. {nameof(this.MAXSIZE)}:{this.MAXSIZE}");
```

Comment:
haven't noticed before. We are using EventSource verbose messages for this king of messages.

Reply:
I will remove it because it's not needed anymore.